src/implementation.ts

```diff
@@ -80,120 +80,8 @@ function callResource(resourceConfig: ResourceConfig, resourcePath: ReadonlyArra
     `;
 }
 
-function getBatchLoader(resourceConfig: BatchResourceConfig, resourcePath: ReadonlyArray<string>) {
-    assert(
-        resourceConfig.isBatchResource === true,
-        `${errorPrefix(resourcePath)} Expected getBatchLoader to be called with a batch resource config`,
-    );
-    assert(
-        typeof resourceConfig.batchKey === 'string' && typeof resourceConfig.newKey === 'string',
-        `${errorPrefix(resourcePath)} Expected both batchKey and newKey for a batch resource`,
-    );
```

Comment:
do we want to keep this?

Reply:
These are kept, see https://github.com/Yelp/dataloader-codegen/pull/329/files/f3649a170a72e6c540eb99e90183b495253fe038#diff-95b044cae616e074c57cf0ac8c92033f207ad57c9c03ef1800d24d1bde85e302R313-R321